core/src/test/scala/unit/kafka/server/AbstractFetcherManagerTest.scala

```diff
@@ -20,7 +20,7 @@ import com.yammer.metrics.core.Gauge
 import kafka.cluster.BrokerEndPoint
 import kafka.metrics.KafkaYammerMetrics
 import kafka.utils.TestUtils
-import org.apache.kafka.common.TopicPartition
+import org.apache.kafka.common.{TopicPartition, Uuid}
```

Comment:
Should we add few unit tests for the newly added methods in this suite and in `AbstractFetcherThreadTest` ?